content/participate/report-issue.adoc

```diff
@@ -31,9 +31,8 @@ functionality should be added.
 
 Other types (_Patch_, _Task_) are not used in JIRA.
 Neither will be explained here.
-If you want to propose a patch, create an issue using one of the 3 types above, and then create a link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[pull request in GitHub].
-be explained here. If you want to propose a patch, create an issue of
-one of the 3 types above and then create a link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[pull request in GitHub].
+If you want to propose a patch, create an issue using one of the 3 types above, and then create a pull request on GitHub link:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request[as explained here].
+. Report issues link:https://github.com/jenkins-infra/helpdesk/issues/[here] that are not related to core Jenkins, its plugins, or security. 
```

Comment:
```suggestion
. Report issues with infrastructure hosted by the Jenkins project (jenkins.io, issues.jenkins.io, etc.) link:https://github.com/jenkins-infra/helpdesk/issues/[here].
```
Thoughts? Core, plugins and security feels pretty widespread to me 🤔

Reply:
Agree.

I would also replace "here" by "the dedicated help desk" or something like that, more descriptive.

Reply:
Sure @lemeurherve @NotMyFault
I have made the above changes!